specs/default/cluster-init/files/azure_centos_7.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2016-2021 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
```

Comment:
We are not really using this file (i think). since in the scripts section we are actually cloning another  repo.

Reply:
yes will remove this file and add in readme required changes for dev testing